internal/services/containers/container_registry_resource.go

```diff
@@ -1133,28 +1178,6 @@ func resourceContainerRegistrySchema() map[string]*pluginsdk.Schema {
 							},
 						},
 					},
-
-					"virtual_network": {
```

Comment:
@magodo this is a breaking change/regression - was this intentionally removed?

Reply:
@tombuildsstuff Yes, this is intentionally. From the service team:

> We deliberatively deprecated this property since it is only used exclusively for service endpoints (which is a feature we are deprecating).  We are encouraging customers to use Private Endpoints instead.

Reply:
Where's the notice for this for users? This is a breaking change so should have waited until 4.0 - unfortunately this is a regression (which has been reported in #18232), so we'll need to reintroduce this, can you send a PR for that?

The ARM Team have previously stated that Service Teams should be giving 3 years notice for deprecations - [whilst there's an open PR to make deprecation messaging available in the API in future](https://github.com/microsoft/api-guidelines/issues/386) - within Terraform this functionality should stick around until 4.0 where we can remove it.

Reply:
@magodo since this is a regression we've reverted this PR in https://github.com/hashicorp/terraform-provider-azurerm/pull/18239 - we can look to support these new fields in the future, but since this would be a breaking change support for this will need to kick around until 4.0.